chain/rescan.go

```diff
@@ -0,0 +1,42 @@
+package chain
+
+import "github.com/lightninglabs/neutrino"
+
+var _ rescanner = (*neutrino.Rescan)(nil)
+
+// updater is the interface that wraps the Update method of a rescan object.
+
+// Update targets a long-running rescan/notification client with updateable
```

Comment:
The Godoc comment for the `Update` method should be right above the method definition. So it's duplicated now, can just move this to replace what's on L14. Same for the `Start()` method of the `starter`.